fhi_users/auth/auth_forms.py

```
@@ -53,3 +53,40 @@ class ChangePasswordForm(forms.Form):
+        required=True, widget=forms.PasswordInput, help_text="Confirm your password"
+    )
+    first_name = forms.CharField(required=False, help_text="Optional")
+    last_name = forms.CharField(required=False, help_text="Optional")
+
+    def clean(self):
+        cleaned_data = super().clean()
+        password = cleaned_data.get("password")
+        confirm_password = cleaned_data.get("confirm_password")
+
+        if password and confirm_password and password != confirm_password:
+            raise forms.ValidationError("Passwords do not match")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)